src/Timestampable/Mapping/Driver/Yaml.php

```diff
@@ -59,9 +59,9 @@ public function readExtendedMetadata($meta, array &$config)
         $mapping = $this->_getMapping($meta->getName());
 
         if (isset($mapping['fields'])) {
-            foreach ($mapping['fields'] as $field => $fieldMapping) {
+            foreach (($mapping->fields ?? $mapping['fields']) as $field => $fieldMapping) {
                 if (isset($fieldMapping['gedmo']['timestampable'])) {
-                    $mappingProperty = $fieldMapping['gedmo']['timestampable'];
+                    $mappingProperty = ($fieldMapping->gedmo ?? $fieldMapping['gedmo'])['timestampable'];
```

Comment:
Another method that can be reverted since it's parsing the raw array straight from YAML (the other change in this file is fine).